test/quantization/fx/test_quantize_fx.py

```diff
@@ -5795,6 +5797,10 @@ def test_linear_tanh_lowering(self):
         """
         from torch.ao.quantization.backend_config import get_onednn_backend_config
         qconfig_mapping = get_default_qconfig_mapping('onednn')
+        # It's required that separate ops have the same qconfig in a fused op
+        qconfig = get_default_qconfig('onednn')
+        qconfig_mapping.set_object_type(torch.nn.Linear, qconfig)
+        qconfig_mapping.set_object_type(torch.nn.Tanh, qconfig)
```

Comment:
why is this not part of get_default_qconfig_mapping?

Reply:
I see, I just saw the summary message

Reply:
also please add TODO comment to all callsites that is using this, we want to refactor them later when we have new api ready

Reply:
Ok It's added.